doc/superbuild/source/developers/releases/Checklist.rst

```diff
@@ -73,6 +73,7 @@ web pages.
    that they are deprecated).
 
 #. Regenerate the Fortran 2003 interfaces. It is possible nothing will be updated.
+   This is done by running ``make all32 all64`` in the ``swig/`` directory.
```

Comment:
It looks like a simple `make` or even `make all` would also suffice, since you have the first explicit target 
```
all: all32 all64
```

Reply:
I thought so too but I could not get it to work. If you happen to have an idea on how to fix it, it would be appreciated.

Reply:
When you run `make`  or `make all` it only runs the all32 target.